src/pages/NervosDao/DaoTransactions.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react'
+import browserHistory from '../../routes/history'
+import { AppContext } from '../../contexts/providers'
+import TransactionItem from '../../components/TransactionItem'
+import { TransactionsPagition } from './styled'
+import Pagination from '../../components/Pagination'
+import { PageParams } from '../../utils/const'
+
+export default ({ currentPage = 1, pageSize = PageParams.PageSize }: { currentPage?: number; pageSize?: number }) => {
+  const { nervosDaoState } = useContext(AppContext)
+  const { transactions, total } = nervosDaoState
+
+  const totalPages = Math.ceil(total / pageSize)
+
+  const onChange = (page: number) => {
```

Comment:
This method could be moved outside the component.

Reply:
`pageSize` come from component props, and `onChange` method only has one paramether.

Reply:
I see.